botorch/settings.py

```diff
@@ -112,6 +112,10 @@ def _set_level(cls, level: int) -> None:
         logger.setLevel(level)
 
     def __init__(self, level: int = LOG_LEVEL_DEFAULT) -> None:
+        """
+        Args:
+            level: How much to log. Defaults to LOG_LEVEL_DEFAULT.
```

Comment:
https://www.youtube.com/watch?v=FshU58nI0Ts

```suggestion
        r"""
        Args:
            level: The log level. Defaults to LOG_LEVEL_DEFAULT.
```